src/main/java/seedu/address/model/recipe/attribute/InstructionList.java

```diff
@@ -10,7 +10,7 @@
  * Represents a Recipe's instructions in the recipe book.
  */
 public class InstructionList {
-    private final ArrayList<Instruction> instructionData;
+    public final ArrayList<Instruction> instructionData;
```

Comment:
Why is this public? Instance variables should always be private; maybe declare and define an instance method, `List<Instruction> asList()` instead of exposing the variable?

Reply:
the other classes define their fields as public.